Userland/Shell/Builtin.cpp

```diff
@@ -149,6 +149,20 @@ ErrorOr<int> Shell::builtin_where(Main::Arguments arguments)
     return at_least_one_succeded ? 0 : 1;
 }
 
+ErrorOr<int> Shell::builtin_reset(Main::Arguments)
```

Comment:
This stuff is a bit odd, as I don't think `reset` actually does any of this on any platform I know, but the builtin looks useful so it's probably fine.